app/lib/forms/choose_your_npq.rb

```diff
@@ -17,8 +17,8 @@ def next_step
       if changing_answer?
         if no_answers_will_change?
           :check_answers
-        elsif course.aso?
-          :about_aso
```

Comment:
This page doesn't exist and nobody will be able to register for EHCOs, this should've been swappe over already.